File: Coders/CursoRegex/conjuntos/intervaloCuidados.js
const texto = 'ABC [abc] a-c'

console.log(texto.match(/[a-z]/gi))
console.log(texto.match(/a-c/g))
console.log(texto.match(/[A-z]/g))
console.log(texto.match(/[Z-a]/g))
// console.log(texto.match(/[a-Z]/g)) erro de sintaxe

File: coders/curso_regex/conjuntos/intervaloCuidados.py
import re

texto = 'ABC [abc] a-c'

print(re.findall(r'[a-z]', texto, re.IGNORECASE))
print(re.findall(r'a-c', texto))
print(re.findall(r'[A-z]', texto))
print(re.findall(r'[Z-a]', texto))
# print(re.findall(r'[a-Z]', texto)) erro de sintaxe
